taker-frontend/src/App.tsx

```diff
@@ -265,6 +279,18 @@ function HistoryLayout({ cfds, connectedToMaker }: HistoryLayoutProps) {
     </VStack>);
 }
 
+function PromoBanner() {
+    return (<HStack>
+        <Center>
+            <Alert status="info">
+                <Link href="http://testing.itchysats.network/" isExternal>
+                    🎁Celebrating pitching on Bitcoin2022 in Miami... 🎉
```

Comment:
Is this correct? 

```suggestion
                    🎁Celebrating pitching at Bitcoin2022 in Miami... 🎉
```

Reply:
thank you, it was indeed a grammar error!